descriptors/ModuleDescriptor-template.json

```diff
@@ -45,7 +45,8 @@
             "users.item.get",
             "converter-storage.field-protection-settings.get",
             "instance-authority-links.instances.collection.get",
-            "instance-authority.linking-rules.collection.get"
+            "instance-authority.linking-rules.collection.get",
+            "linking-rules.instance-authority.get"
```

Comment:
Do we need it on put also? Or it's not used there?

Reply:
I think it will be good to check and to fix all permissions issues in one commit

Reply:
Also, I've found that `linking-rules.instance-authority.get` is not exist permission. The real one is `instance-authority.linking-rules.collection.get` and it's already in place. So, this hasn't to be merged.

Reply:
yes